src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndRecommendations.js

```diff
@@ -143,6 +191,156 @@ const DiagnosisAndRecommendations = ({
     }
     return clean;
   };
+
+  const onSubmit = async (data) => {
+    if (isRequiredValidation(data)) {
+      try {
+        const APIsArray = [];
+        const items = qItem.item.map((i) => {
+          const item = {
+            linkId: i.linkId,
+            text: i.text,
+          };
+          switch (i.text) {
```

Comment:
the switch needs to be on the linkId, the text can be changed. it's better to put the text in a comment then the code will be more readable